SendMessage/index.test.js

```diff
@@ -4,7 +4,11 @@ const { mockNotifyClient } = require('notifications-node-client').mocks
 const context = require('../test/defaultContext')
 const testEnvVars = require('../test/testEnvVars')
 
-const sendMessage = require('./index')
+const sendMessage = require('.')
```

Comment:
Rather than use the `__mocks__` manual mocks facility in Jest I find it cleaner to use `jest.mock` to keep the mocking code nearer the test. For example in this file the lines:
```
const { mockSendSms } = require('notifications-node-client').mocks
const { mockNotifyClient } = require('notifications-node-client').mocks
```
and the associated `__mocks__` file could be replaced with:
```
const mockSendSms = jest.fn()

jest.mock('notifications-node-client', () => {
  return {
    NotifyClient: jest.fn().mockImplementation(() => {
      return { sendSms: mockSendSms }
    })
  }
})

const { NotifyClient } = require('notifications-node-client')
```

in that case `mockNotifyClient` would be replaced with `NotifyClient`

Reply:
Agreed. I'm using a similar technique is another app. I'll refactor this once I've finished that.